docker/raspberry_pi.md

```diff
@@ -0,0 +1,94 @@
+# How to use Orion Context Broker on Raspberry Pi.
```

Comment:
So they idea is to install docker in the Rasberry PI operating system, then build and run Orion as docker container on that system, isn't it? (vs. build and run Orion directly in the Raspberry PI operating system)

Just to check if I understand correctly :)

Reply:
Your understanding is correct. It build and run Orion as docker container on a Raspberry Pi.

Reply:
Ok. Thanks for the clarification!

NTC